frontend/express/public/javascripts/countly/countly.vue.components.js

```diff
@@ -372,4 +373,2886 @@
+                                    </div>\
+                                </div>\
+                                <div class="drop combo"></div>\
+                            </div>\
+                            <div class="search" v-if="searchable" v-show="opened">\
+                                <div class="inner">\
+                                <input type="search"\
+                                    @focus="hasFocus = true"\
+                                    v-model="tempSearchQuery"/>\
+                                <i class="fa fa-search"></i>\
+                                </div>\
+                            </div>\
+                            <div class="items-list square" v-show="opened">\
+                                <div ref="scrollable" class="scrollable">\
+                                    <div class="warning" v-if="dynamicItems && listDelayWarning">{{ listDelayWarning }}</div>\
+                                    <div v-for="(item, i) in visibleItems" :key="i"\
+                                        @mouseover="setNavigatedIndex(i)"\
+                                        @mouseleave="setNavigatedIndex(null)"\
+                                        @click="setItem(item)"\
+                                        :ref="\'tmpItemRef_\' + i"\
+                                        :class="{item: !isItemGroup(item), group : isItemGroup(item), navigated: i === navigatedIndex}">\
+                                        <div v-if="isItemGroup(item)">\
+                                            <span v-text="item.name"></span>\
+                                        </div>\
+                                        <div v-else v-bind:data-value="item.value">\
+                                            <span v-text="item.name"></span>\
+                                        </div>\
+                                    </div>\
+                                </div>\
+                            </div>\
+                        </div>'
+        }
+    ));
+
+    Vue.component("cly-back-link", countlyBaseComponent.extend(
+        // @vue/component
+        {
+            mixins: [
+                _mixins.i18n
+            ],
+            props: {
+                title: {type: String, required: false},
+                link: {type: String, required: false}
+            },
+            methods: {
+                back: function() {
+                    if (this.link) {
+                        app.back(this.link);
+                    }
+                    else {
+                        app.back();
+                    }
+                }
+            },
+            computed: {
+                innerTitle: function() {
+                    if (this.title) {
+                        return this.title;
+                    }
+                    return this.i18n("common.back");
+                }
+            },
```

Comment:
Codacy found an issue: [Multiline support is limited to browsers supporting ES5 only.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)